src/WalletFramework.Oid4Vc/Oid4Vp/AuthResponse/Encryption/EncryptedAuthorizationResponse.cs

```
@@ -44,20 +51,20 @@ public static EncryptedAuthorizationResponse Encrypt(
         var settings = new JwtSettings();
         settings.RegisterJwe(JweEncryption.A256GCM, new AesGcmEncryption());
 
+        var selectedEncAlgorithm = encryptedResponseEncAlgorithms.Match(
+            encAlgs => encAlgs.FirstOrDefault(encAlg => SupportedEncAlgorithmsMap.ContainsKey(encAlg)) 
+                    ?? throw new NotSupportedException("Unsupported response encryption algorithms requested by verifier."),
+            () => DefaultEncAlgorithm);
```

Why is the default alg chosen here ? Shouldnt we return an error that its not supported ?

---

The encryptedResponseEncAlgorithms is optional and A128GCM is the default according to the spec